config/types/file.go

```diff
@@ -24,13 +24,18 @@ var (
 	ErrFileIllegalMode = errors.New("illegal file mode")
 )
 
+// File represents all common info for files (special types, e.g. directories, included).
```

Comment:
`Node` is more appropriate than `File` for this, since it very much relates to the `inode`-associated context.

Probably makes sense to `s/File(Mode|User|Group)/Node\1/g` as well...